scripts/upgrade

```diff
@@ -356,10 +356,9 @@ yunohost service add "gitlab-runsvdir" --log "/var/log/$app/gitlab-rails/applica
 # SETUP SSOWAT
 #=================================================
 
-# If app is public, add url to SSOWat conf as skipped_uris
+# Make app public if necessary
 if [ $is_public -eq 1 ]; then
```

Comment:
Note that here, you are **not** supposed to touching the permission state anymore ... In fact, you shouldn't even care about the value of `$is_public` at all. Because since the app got installed, maybe the admin changed the permissions from "visitors" to "all_users" or to a bunch of more complex permissions ...

So in my opinion, what should be done is : 
- in the install script, don't store anymore the `is_public`. You could argue that it doesn't hurt to store it, yet still, it adds confusion if somebody who does not understand what it corresponds to encounters it and is not aware of the new permission system (which won't be so 'new' anymore in 1 year) ... (and it's not as if we regularly see users attempting to tweak the settings.yml hoping it magically propagates to config files or whatever) ;
- in the upgrade script, you don't have to do any migration - except possibly delete that legacy "is_public" setting if it still exists - because migrating to the new permission system will be done automatically by the core according to the value of the `skipped`/`unprotected_uris` setting ;
- more generally, my expectation is that, in the upgrade script, you should never have to update the permissions **except possibly** because you want to define some new specific / custom permission for example to manage the access to some admin interface or whatever

Reply:
Thank you for your feedback! I'll update that. (And I'll start modifying example and "official" apps)